Add features and update functional

krab-sergeevich opened the pull request:
add result.csv to gitignore
create dump_result_in_csv

krab-sergeevich committed:
create dump_result_in_csv
add result.csv to gitignore

krab-sergeevich committed:
change max length from 120 to 140
add logging

davdeenko commented on .gitignore:
```diff
@@ -1,2 +1,13 @@
 /.idea/
 /venv/
+/result1.csv
```
Давай использовать /result*.csv вместо дублирования файлов

davdeenko commented on set.pylintrc:
```diff
@@ -1,2 +1,2 @@
 [FORMAT]
-max-line-length=120
\ No newline at end of file
+max-line-length=140
```
Вот так делать не надо. По пеп8 надо использовать 79 символов. Но это ограничение для старых экранов, сейчас стандарт в индустрии - 120, но не больше. Если не помещается в одну строку - надо декомпозировать логику

davdeenko commented on main.py:
```diff
@@ -45,15 +55,24 @@ def dump_result_in_pandas(name_data_color_size_local: list[list[str]], reviews_o
-    name_date_color_size = [i.text.split('\n') for i in browser.find_elements(By.CLASS_NAME, "feedback__info")]
-    reviews_of_customers = [i.text for i in browser.find_elements(By.CLASS_NAME, "feedback__text")]
-    print(dump_result_in_pandas(name_date_color_size, reviews_of_customers))
+    list_of_links = ['https://www.wildberries.ru/catalog/21659599/feedbacks?imtId=10317392',
```
Давай будем читать их из другого файла со списком ссылок на товары

davdeenko commented on main.py:
```diff
@@ -45,15 +55,24 @@ def dump_result_in_pandas(name_data_color_size_local: list[list[str]], reviews_o
+        browser.get(link)
+        logging.info("browset get link")
+        scroll_page(browser)
+        name_date_color_size = [i.text.split('\n') for i in browser.find_elements(By.CLASS_NAME, "feedback__info")]
```
давай использовать что-нибудь осмысленное вместо i

krab-sergeevich committed:
change max length from 140 to 120
change gitignore
add annotations
Change name of parameters and arguments of functions and names of variables

krab-sergeevich committed:
create product_review_links
add reading links from product_review_links
